src/TeamStore.Keeper/Interfaces/IAssetService.cs

```diff
@@ -18,6 +18,8 @@ public interface IAssetService
 
         Task<List<Asset>> GetAssetsAsync(int projectId);
 
+        Task<List<AssetSearchResult>> GetAssetResultsAsync();
```

Comment:
It is never a good design to deal with more data than needed. A key principle in being efficient is to only work with what you need. This uses unnecessary memory, data transfer and decrypting CPU power, which is expensive.